src/libraries/System.IO.Compression/src/System/IO/Compression/ZipBlocks.cs

```diff
@@ -534,6 +568,24 @@ public static bool TryValidateBlock(BinaryReader reader, ZipArchiveEntry entry)
 
             return true;
         }
+
+        private static ulong ConvertToUlong(uint lowerAddressValue, uint higherAddressValue)
+        {
+            return (ulong)lowerAddressValue + (((ulong)higherAddressValue) << 32);
+        }
+
+        private static bool TestMatch(ZipArchiveEntry entry, uint suspectSignature, uint suspectCrc32, ulong suspectCompressedSize, ulong suspectUncompressedSize)
+        {
+            if (DataDescriptorSignature == suspectSignature && (ulong)entry.CompressedLength == suspectCompressedSize
```

Comment:
Should the arguments be also called "*Length" to match the property names that they are compare to?

Reply:
Sure